pulp/mps_lp.py

```diff
@@ -142,12 +151,20 @@ def readMPS(path, sense, dropConsNames=False):
                 readMPSSetBounds(line, variable_info)
                 if line[1] not in bnd_names:
                     bnd_names.append(line[1])
+            elif mode == CORE_FILE_OBJSENSE_MODE:
+                parameters['sense'] = const.LpMaximize if line[0] == 'MAX'\
+                        else const.LpMinimize
```

Comment:
shouldn't we test if `parameters['sense']` is None? or if we provided an argument to the function (i.e., via the `sense` argument?

Reply:
We could put in a check here for whether `parameters['sense']` is `None`. However, in the current code the check is not necessary since it is only possible to enter the `CORE_FILE_OBJSENSE_MODE` mode if `parameters['sense']` is `None`.

We could put in an assert or throw an exception to protect against future code changes? Something like (after line 154)
```
if parameters['sense'] is not None
    raise Exception("The supplied objective sense will be overwritten by the MPS file objective sense")
```